orchagent/portsorch.h

```diff
@@ -30,6 +30,7 @@ class PortsOrch : public Orch, public Subject
 
     bool isInitDone();
 
+    map<string, Port>& getAllPorts();
     bool getPort(string alias, Port &port);
     bool getPort(sai_object_id_t id, Port &port);
```

Comment:
In some places I noticed that getPort(oid, &port) is used for oid values obtained from XXX_BRIDGE_PORT_ID attribute,
it looks for me that it is needed to make additional check for sai_object_type_query(oid) == SAI_OBJECT_TYPE_BRIDGE_PORT and match oid parameter by "port.m_bridge_port_id" ?